src/components/BlogCard/BlogCard.examples.md

```diff
@@ -0,0 +1,16 @@
+```jsx
+<BlogCard
+  aside
+  title=""
+  titleUrl=""
```

Comment:
Let's just add in some real values here to some of these fields.  Then we can add this component to the documentation config file and regenerate the documentation.